apps/civicsignalblog/src/payload/collections/Research/Pages.js

```diff
@@ -0,0 +1,31 @@
+import canRead from "../../access/applications/research";
+import CustomPageHeader from "../../blocks/CustomPageHeader";
+import Error from "../../blocks/Error";
+import FeaturedStories from "../../blocks/FeaturedStories";
+import LongForm from "../../blocks/LongForm";
+import PageHeader from "../../blocks/PageHeader";
+import Posts from "../../blocks/Posts";
+import pages from "../../utils/createPagesCollection";
+
+const Pages = pages({
+  pageSlug: "pages",
```

Comment:
Minor and hence can be ignored or implemented in another PR but I think it makes a bit more sense to migrate `pageSlug` from `pages` to `${RESEARCH}-pages` now when we have have very few pages in PROD

Reply:
✅ 

I've done it now, I think we'll  have to recreate the pages when we migrate to prod, but its a very minor issue

Reply:
...or a [migration](https://payloadcms.com/docs/database/migrations) script instead of recreating them.